internal/test/e2e/e2e_test.go

```diff
@@ -212,4 +213,24 @@ func runAllTests(t *testing.T, kubectl *kubectlContext) {
 	}, time.Minute*2, time.Second*10)
 	require.NoError(t, err)
 	t.Log("ETCD is reachable from host machine")
+
+	err = try.Eventually(func() error {
+		t.Log("Checking if the EtcdCluster resource has the correct status")
+		members, err := kubectl.Get("--namespace", "default", "etcdcluster", "my-cluster", "-o=jsonpath='{.status.members...name}'")
+
+		if err != nil {
+			return err
+		}
+		for _, expectedMemberName := range []string{
+			"my-cluster-0",
+			"my-cluster-1",
+			"my-cluster-2",
```

Comment:
i think we should just be checking that we have 3 items in `status.members` - otherwise we couple this test to the naming of members, which is something the end user shouldn't care about.